analysis/preview.py

```diff
@@ -223,19 +239,25 @@ def main():
     args = parse_args()
     jdoc = get_configuration(args.input_file)
 
-    zsps, drones = parse_positions(jdoc)
+    zsps, drones, step = parse_positions(jdoc)
 
     ax = setup_3dplot()
     build_zsp_space(ax, zsps)
 
     trajectories = []
+    i=0
```

Comment:
[`enumerate()`](https://docs.python.org/3/library/functions.html?highlight=enumerate#enumerate) allows to make the for loop cleaner, without the outside `i` index variable